src/target/linux.rs

```diff
@@ -306,6 +306,14 @@ impl LinuxTarget {
         )
     }
 
+    /// unmap the memory.
+    pub fn munmap(
+        &self,
+        addr: *mut libc::c_void,
+        length: libc::size_t,
+    ) -> CrabResult<libc::c_ulonglong> {
+        self.syscall(libc::SYS_munmap as _, addr as _, length as _, 0, 0, 0, 0)
+    }
```

Comment:
Very minor nitpick – could you please add a missing newline after the closing curly bracket?

```suggestion
    }

```